exercise_02/callbacks/onSuccess.js

```diff
@@ -8,3 +8,13 @@
  * output:
  *  - string: User with email {email} has been correctly created
  */
+
+/**
+ * It receives an email as string and returns a string
+ * @param {string} email
+ * @returns {string}
+ */
+function onSuccess(email) {
+  return `User with email ${email} has been correctly created`;
+}
+export default onSuccess;
```

Comment:
Improve Code Readability with Clearer Code Blocks. Fix it!
```suggestion

export default onSuccess;
```